qubes/vm/qubesvm.py

```diff
@@ -2533,16 +2533,17 @@ def kernelopts_common(self):
         """
         if not self.kernel:
             return ""
+        base_kernelopts = "systemd.machine_id=" + self.uuid.hex + " "
```

Comment:
Can you at least guard it with `if self.features.check_with_template("os") == "Linux"`? And/or make it possible to disable some other way?

Reply:
I’ll do the first for now, and figure out the rest once Qubes OS supports a distro (like Alpine) that is not systemd-based.

Reply:
AFAIK there is a community Alpine template

Reply:
Fair, but let’s get this merged first for Whonix.